fontforgeexe/startui.c

```diff
@@ -614,7 +624,11 @@ static int splash_e_h(GWindow gw, GEvent *event) {
 	GDrawSetFont(gw,splash_font);
 	y = splashimage.u.image->height + as + fh/2;
 	for ( i=1; i<linecnt; ++i ) {
-	    if ( is>=lines[i-1]+1 && is<lines[i] ) {
+        if (i == 11) {
```

Comment:
Can this 11 be expressed more symbolically? Not in the sense of a `#define` for its own sake, but could there be a define (or whatever) for this expressed in more abstract terms? How do we know it's 11 and can that knowledge be expressed in other terms?

Reply:
Well the way lines are added is a bit of a mess. Anyone who wants to dramatically change the text is going to have to deal with this. I can put a `#define` and a comment about how to compute it if you think it would help though

Reply:
Even just a comment explaining where it comes from and under what circumstances it would need to change would be sufficient, I think. I'm old fashioned in the way that worries about specific numbers appearing in if guards without some tie to what motivates them.